### DISCUSSION TITLE
一位中介关于“正确方法论”的回应

### DISCUSSION THREAD

**Discussion by ray999** [The-Run-Philosophy-Organization/run#319] [General]
http://xhslink.com/NqMH3g

**Comment by ray999** [comment]
这是小红书上一位中介发的视频，针对本库中的某篇文章提到内容进行的回应。

**Comment by the-run-philosophy** [comment]
@ray999 回应的具体内容能说一下吗

**Reply by ray999** [reply]
有点长，具体可以看着这个小红书的链接吧（放心戳）。  http://xhslink.com/NqMH3g
